pkgs/activitywatch/commit-hash.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/vue.config.js b/vue.config.js
+index 02c0699..0c4a014 100644
+--- a/vue.config.js
++++ b/vue.config.js
+@@ -4,10 +4,7 @@ const CopyWebpackPlugin = require('copy-webpack-plugin');
+ const { argv } = require('yargs');
+ 
+ // get git info from command line
+-const _COMMIT_HASH = require('child_process')
+-  .execSync('git rev-parse --short HEAD')
+-  .toString()
+-  .trim();
++const _COMMIT_HASH = "EMPTY";
```

Comment:
I would probably use `@COMMIT_HASH@` and then use

```
  patches = [
    # Hardcode version to avoid the need to have the Git repo available at build time.
    (substituteAll {
      src = ./commit-hash.patch;
      COMMIT_HASH = sources.rev;
    })

Reply:
Unfortunately it'll simply put the version of the package and not the actual commit hash, but it's probably better than nothing